ansible/playbooks/adhoc/grow_root_vg/grow_root_vg.yml

```diff
@@ -0,0 +1,132 @@
+---
+# This playbook grows the root VG on a node by:
+#  * expanding /dev/xvda
+#  * using growpart to expand the rootvg partition
+#  * grow /var to 64GB
+#
+#  To run:
+#  1. Source your AWS credentials (make sure it's the corresponding AWS account) into your environment
+#    export AWS_PROFILE=<aws account>
+#    export AWS_DEFAULT_REGION=<aws region>
+#
+# 2. run the playbook:
+#   ansible-playbook -e 'cli_tag_name=<tag-name>' grow_root_vg.yml
+#
+#  Example:
+#   ansible-playbook -e 'cli_tag_name=ops-compute-12345' grow_root_vg.yml
+#
+#  Notes:
+#  * By default this will do a900GB GP2 volume.  The can be overidden with the "-e 'cli_volume_size=something'" variable
+#  * This can be done with NO downtime on the host, but you probably ought to test a reboot 
+#
+
+- name: Grow the rootvg volume group
+  hosts: "oo_name_{{ cli_tag_name }}"
+  user: root
+  connection: ssh
+  gather_facts: no
+
+  vars:
+    cli_volume_size: 90
+    cli_var_size: 64
```

Comment:
There appears to be a relationship between these two sizes.  Do we need to check that `cli_volume_size` - `cli_var_size` >= `<SOME MINIMUM>`?

Reply:
added this logic.